pyomo/contrib/fbbt/fbbt.py

```diff
@@ -727,6 +733,8 @@ def visiting_potential_leaf(self, node):
                     lb = -math.inf
                 if ub is None:
                     ub = math.inf
+                if lb - self.infeasibility_tol > ub:
+                    raise InfeasibleConstraintException('Variable has a lower bound which is larger than its upper bound: {0}'.format(str(node)))
             self.bnds_dict[node] = (lb, ub)
             return True, None
```

Comment:
Out of curiosity, I think `LinearExpression` still presents itself as a leaf node (see https://github.com/Pyomo/pyomo/issues/924). Do we have some kind of handling for that situation?

Reply:
No, ```LinearExpression``` is not currently supported. I will have to add support in another PR.

Reply:
@michaelbynum Have you added an issue about that already?

Reply:
@carldlaird Yes, there is an existing issue. I just updated it too.